### PR TITLE
SQL production

### PR DESCRIPTION
**Pull request by rareddy**
Adding the Json based result production such that nested and sibling nodes in graphql query can be processed with a single query. JOOQ support is needed for this to work with multiple databases

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,15 +33,14 @@
...
     <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-    <quarkus.platform.version>1.5.2.Final</quarkus.platform.version>
+    <quarkus.platform.version>1.7.0.Final</quarkus.platform.version>
     <surefire-plugin.version>2.22.1</surefire-plugin.version>
     <failsafe-plugin.version>3.0.0-M3</failsafe-plugin.version>
     <version.com.graphql-java>15.0</version.com.graphql-java>
-    <version.org.postgresql>42.2.14</version.org.postgresql> 
-    <version.org.teiid>15.0.0</version.org.teiid>
+    <version.org.jooq>3.14.0-SNAPSHOT</version.org.jooq>
```
Comment (ani-sha) [inline pom.xml:43]: Facing dependency issuses for `3.14.0-SNAPSHOT` version.

Comment (rareddy) [reply pom.xml:43]: yes, it depends on new version of that library yet to be released, so for now clone jooq.org project locally and run

**Comment by wtrocki** [comment]
going to review and build this from master now.

**Comment by machi1990** [comment]
> going to review and build this from master now.

We'll need jooq release right? I am seeing a snapshot version is in use.

**Comment by rareddy** [comment]
yes, currently we require 3.14 version of JOOQ which is not released yet, I have used the SNAPSHOT, for now, to check you would need to download their repo and build locally to succeed

**Comment by wtrocki** [comment]
Yep. Came here to mention the same problem I had. Any ETA on the JOOQ ?

**Comment by rareddy** [comment]
With their past release cadence seem to be very soon, maybe couple weeks.

**Pull request merged**